flow-server/src/test/java/com/vaadin/flow/router/RouterTest.java

```diff
@@ -2664,6 +2696,25 @@ public void custom_exception_target_should_override_default_ones() {
                 EXCEPTION_TEXT);
     }
 
+    @Test
+    public void custom_access_denied_exception_target_should_override_default_ones() {
+        setNavigationTargets(FailOnAccessDeniedException.class);
+        setErrorNavigationTargets(NonExtendingAccessDeniedTarget.class,
+                RouteNotFoundError.class);
```

Comment:
```suggestion
                RouteAccessDeniedError.class);
```
This should be the access denied default view so we see that it the custom one overrides the default